source/core/coverage/ut_coverage_helper.pkb

```diff
@@ -59,17 +68,18 @@ create or replace package body ut_coverage_helper is
   end;
 
   procedure coverage_stop is
-    l_return_code binary_integer;
   begin
     if not g_develop_mode then
```

Comment:
No, not really.
We need to override `coverage_stop` in out unit tests (for now), so that when we test the framework, our "develop-mode" coverage will not stop, if we test execution for one of coverage reporters.

We will be able to remove the develop mode, once we move to testing ut3 with another ut3 installation. 

It needs to stay like this for now, otherwise we will not get a true picture of coverage for out own code.